docs/guides/implementation-guides/how-to-implement-dependency-injection.md

```diff
@@ -14,11 +14,21 @@ Let's assume that your MainViewModel has a dependency on MyService. The construc
 ```csharp
 private readonly MyService _myService;
 
-public MapsViewModel(MyService myService)
+public MainViewModel(MyService myService)
 {
     _myService = myService;
 }
 ```
+
+You could solve that by instanciating `MyService` and passing as a parameter everytime you need a `MainViewModel` instance, like this:
+```csharp
+var window = new MainWindow
+{
+    DataContext = new MainViewModel(new MyService())
+}
+```
+but this will get quickly difficult as when your application scale and your classes have more dependencies, hence the need of DI.
```

Comment:
Also for some reason Github seems to think this is C#, you may not have closed a code block properly? I can't see anything obvious though.

Reply:
I've jump line before and after the code block like I've done for the ones in the page. Hopefully it should fix it